gapic/src/main/com/google/gapid/perfetto/canvas/PanelGroup.java

```diff
@@ -167,17 +175,31 @@ private Child findPanel(double y) {
   private static class Child {
     public final Panel panel;
     public double y, h;
-    public boolean visible;
+    private boolean visible;
+    private boolean include;
 
     public Child(Panel panel, double y, double h) {
       this.panel = panel;
       this.y = y;
       this.h = Math.max(0, h);
       this.visible = true;
+      this.include = true;
     }
 
     public double getNextY() {
       return y + h;
     }
+
+    public void setVisibile(boolean visible) {
+      this.visible = visible;
+    }
+
+    public void setFiltered(boolean include) {
+      this.include = include;
```

Comment:
Can we rename **include** to **filtered** in this class? It will be consistent with setFiltered then.

Reply:
Done.